contracts/contracts/reward-contract/RewardByBlock.sol

```diff
@@ -39,10 +41,25 @@ contract RewardByBlock is EternalStorage, IRewardByBlock {
         _;
     }
 
+    function initialize(uint256 _blockRewardAmount, address[] calldata _homeBridgeList) external {
```

Comment:
Have you thought about just accepting a single bridge address? Would make the code slightly easier and I don't see a reason why we would need more than one.

Reply:
I'm a bit afraid to change too much of the code without having tests securing the functionality. Having only one bridge touches many more parts of the code.  Having a list but only having one entry do the same. But I can do so. :thinking:

Reply:
This doesn't necessarily require more changes. Just store the single bridge address somewhere, keep `bridgesAllowedLength = 1` and return the address in the `bridgesAllowed` function. This would have the added benefit of not having to remove a public function.

Reply:
Okay, yes this would keep the changes on a minimum. The code look more than ugly, but I will go for it.

Reply:
:heavy_check_mark: